backend/src/model/dto/plantings.rs

```diff
@@ -34,6 +35,12 @@ pub struct PlantingDto {
     /// The y scale of the plant on the map.
     #[serde(rename = "scaleY")]
     pub scale_y: f32,
+    /// The date the planting was added to the map.
+    /// If None, the planting always existed.
```

Comment:
How can you produce such a date? I think we can remove this possibility (make non-null).

If it is only legacy: don't worry, we need to reset the database anyway.

Reply:
My idea was to enable plants that live for the whole selectable timeline. 
Like a tree that was there as long as anyone can remember.
The input in the left toolbar would allow to clear the date such that it can be null.
Default would be the current date.

Reply:
Yes, good idea. It will definitely be needed for other elements in the map (like a hill or river). :wink: